jcl/src/java.base/share/classes/java/lang/Access.java

```diff
@@ -110,6 +110,11 @@ public AnnotationType getAnnotationType(java.lang.Class<?> arg0) {
 	/** Return the constant pool for a class. */
 	public native ConstantPool getConstantPool(java.lang.Class<?> arg0);
 
+	/** Return the constant pool for a class. Use this version to resolve errors from constant pool redefinition. */
+	public ConstantPool getConstantPoolInternal(Object internalConstantPool) {
```

Comment:
We shouldn't add public APIs to java/lang classes as they are not part of the spec. Instead, you can add a helper to com.ibm.oti.vm.VM